src/liquid/components/ld-tabs/ld-tab.css

```diff
@@ -0,0 +1,308 @@
+.ld-tab {
+  --ld-tab-padding-x: 0.875rem;
+  --ld-tab-padding-y: 0.75rem;
+  --ld-tab-min-height: 2.5rem;
+  --ld-tab-color: var(--ld-col-rblck-default);
+  --ld-tab-bg-color: var(--ld-col-bg-lg);
+  --ld-tab-color-focus: var(--ld-col-rblck1);
+  --ld-tab-bg-color-focus: var(--ld-col-rblck4);
+  --ld-tab-bg-color-hover: var(--ld-col-rblck1);
+  --ld-tab-bg-color-active: var(--ld-col-rblck2);
+  --ld-tab-gap: 0.625rem;
+  --ld-tab-icon-size: 1.25rem;
+  position: relative;
+  font: var(--ld-typo-body-m);
+  font-weight: 700;
+  border: 0;
+  text-decoration: none;
+  user-select: none;
+  touch-action: manipulation;
+  display: grid;
+  grid-template-columns: minmax(0rem, var(--ld-sp-16)) max-content minmax(
+      0rem,
+      var(--ld-sp-16)
+    );
+  align-items: center;
+  text-align: center;
+  justify-content: center;
+  line-height: 1;
+  scroll-snap-align: start;
+  white-space: nowrap;
+  -webkit-touch-callout: none;
+  padding: var(--ld-tab-padding-y) var(--ld-tab-padding-x);
+  min-height: var(--ld-tab-min-height);
+  color: var(--ld-tab-color);
+  background-color: var(--ld-tab-bg-color);
+
+  &[aria-disabled='true'] {
+    --ld-tab-bg-color-focus: var(--ld-col-rblck3);
+    --ld-tab-color-focus: var(--ld-col-wht);
+
+    .ld-tab__content {
+      opacity: 0.25;
+    }
+  }
+
+  /* Selection indicator */
+  &::after {
+    content: '';
+    position: absolute;
+    inset: auto 0 0 0;
+    height: var(--ld-sp-2);
+    z-index: 3;
+  }
+
+  &[aria-selected='true'] {
+    color: var(--ld-tab-color-selected);
+
+    &::after {
+      background-color: var(--ld-tab-color-selected);
+    }
+  }
+
+  &:focus:focus-visible {
+    outline: none;
+    background-color: var(--ld-tab-bg-color-focus);
+
+    &:not(:active),
+    &[aria-disabled='true'] {
+      color: var(--ld-tab-color-focus);
+    }
+
+    &[aria-selected='true'] {
+      &:not(:active) {
+        color: var(--ld-tab-selected-focus-color);
+      }
+
+      &::after {
+        background-color: var(--ld-tab-selected-focus-color);
+      }
+    }
+
+    &:not([aria-selected='true'])::after {
+      background-color: inherit;
+    }
+  }
+
+  &:where(:not([aria-disabled='true'])) {
+    cursor: pointer;
+
+    @media (hover: hover) {
+      &:hover {
+        background-color: var(--ld-tab-bg-color-hover);
+
+        &:not([aria-selected='true'])::after {
+          background-color: inherit;
+        }
+      }
+    }
+    &:active:focus,
+    &:active:focus:focus-visible {
```

Comment:
Is there a difference between `:focus:focus-visible` and simply `:focus-visible`?

Reply:
Yes. `:focus-visible` tells the browser to apply the styles if the user is using the keyboard. The element does not need to be focused in this case. `focus:` tells the browser to apply the styles if the element has focus. `:focus:focus-visible` tells the browser to apply the styles if both the element has focus and the user is using the keyboard to navigate the page.